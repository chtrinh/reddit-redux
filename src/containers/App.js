import React, { Component } from 'react';
import { Provider } from 'react-redux';
import configureStore from '../store';
import Reddit from './Reddit';

const store = configureStore();

export default class App extends Component {
  render() {
    return (
      <Provider store={store}>
        <Reddit />
      </Provider>
    );
  }
}
