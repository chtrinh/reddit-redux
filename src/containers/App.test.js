import React from 'react';
import App from './App';

it('renders without crashing', () => {
  const selectedSubreddit = 'frontend';
  const handleChange = jest.fn();

  const wrapper = shallow(<App />);

  expect(wrapper).toMatchSnapshot();
});
