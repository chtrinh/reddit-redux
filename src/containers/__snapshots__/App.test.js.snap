// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
>
  <Connect(Reddit) />
</Provider>
`;
